mlflow/langchain/__init__.py

```diff
@@ -529,10 +531,12 @@ def load_retriever(persist_directory):
                     )
 
             See a complete example in examples/langchain/retrieval_qa_chain.py.
-        example_no_conversion: If ``False``, the input example will be converted to a Pandas
+        example_no_conversion: This parameter is deprecated and will be removed in a future
+                release, please drop it and input examples are not converted anymore.
+                If ``False``, the input example will be converted to a Pandas
                 DataFrame format when saving. This is useful when the model expects a DataFrame
                 input and the input example could be passed directly to the model.
-                Defaults to ``True``.
+                Defaults to ``None``.
```

Comment:
Do we need `If ``False``, the input example will be converted to a Pandas ...`? It sounds like this parameter is still in use.

Reply:
Ah right.. Let me remove it :)